Resources/template.json

```diff
@@ -0,0 +1,222 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "functionAppPrefix": {
+        "type": "string",
+        "metadata": {
+          "description": "Prefix for the function app resources, in the form dfc-env-proj-fnapp"
+        }
+      },
+      "appServicePlanName": {
+        "type": "string",
+        "metadata": {
+          "description": "App service plan to run the function app under"
+        }
+      },
+      "appServicePlanResourceGroup": {
+        "type": "string",
+        "metadata": {
+          "description": "Resource group the app service plan is in"
+        }
+      },
+      "storageConnectionString": {
+        "type": "string",
+        "metadata": {
+          "description": "Connection string for the storage account to store function app files"
+        }
+      }
+    },
+    "variables": {
+      "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/ArmTemplates/",
+      "functionAppName": "[concat(parameters('functionAppPrefix'),'-fa')]",
+      "appInsightName": "[concat(parameters('functionAppPrefix'),'-ai')]"
+    },
+    "resources": [
+        {
+        "apiVersion": "2017-05-10",
+        "name": "[variables('appInsightName')]",
+        "type": "Microsoft.Resources/deployments",
+        "properties": {
+          "mode": "Incremental",
+          "templateLink": {
+            "uri": "[concat(variables('deploymentUrlBase'),'application-insights.json')]",
+            "contentVersion": "1.0.0.0"
+          },
+          "parameters": {
+            "appInsightsName": {
+              "value": "[variables('appInsightName')]"
+            },
+            "attachedService": {
+              "value": "[variables('functionAppName')]"
+            }
+          }
+        }
+        },
+      {
+        "apiVersion": "2017-05-10",
+        "name": "[variables('functionAppName')]",
+        "type": "Microsoft.Resources/deployments",
+       
```

Comment:
Unnecessary white space.  White space should be used consistently to make templates and scripts readable.